src/rpc/ngHandlers/NFTInfo.cpp

```diff
@@ -0,0 +1,133 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2023, the clio developers.
+
+    Permission to use, copy, modify, and distribute this software for any
+    purpose with or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL,  DIRECT,  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <rpc/RPCHelpers.h>
+#include <rpc/ngHandlers/NFTInfo.h>
+
+#include <ripple/app/tx/impl/details/NFTokenUtils.h>
+#include <ripple/protocol/Indexes.h>
+
+using namespace ripple;
+using namespace ::RPC;
+
+namespace RPCng {
+
+NFTInfoHandler::Result
+NFTInfoHandler::process(
+    NFTInfoHandler::Input input,
+    boost::asio::yield_context& yield) const
+{
+    ripple::uint256 tokenID;
+    [[maybe_unused]] auto const _ = tokenID.parseHex(input.nftID);
```

Comment:
ripple::uint256 tokenID{input.nftID.c_str()}; does the same thing.

Reply:
it was crashing.. let me try with `.c_str()`.